### DISCUSSION TITLE
GC9A01 module and back light brightness

### DISCUSSION THREAD

**Discussion by Makuna** [moononournation/Arduino_GFX#31] [Q&A] [answered]
I am using this [module GC9A01 round IPS display module from AliExpress](
https://www.aliexpress.com/item/1005001322032932.html?aff_platform=true&aff_short_key=UneMJZVf&isdl=y&src=bing&albch=shopping&acnt=42005546&isdl=y&aff_short_key=UneMJZVf&albcp=373871275&albag=1308419064071836&slnk=&trgt=pla-4585375808811125&plac=&crea=81776241344382&netw=o&device=c&mtctp=e&utm_source=Bing&utm_medium=shopping&utm_campaign=PA_Bing_customlabel1_US_PC&utm_content=customlabel1%3D7&utm_term=gc9a01&msclkid=37fa195f23ec14778e88796aa00c8580).
I have connected it to Arduino Mega 2560 using the 3.3v to the VCC.  It basically works but I see flash when the brightness is normal and then it is faded and even goes darker toward the bottom of the screen.  
I do have the BLK wire connected, but I went ahead and disconnected (the default is black light on) and displays the same.
Anyone have experience with one of these have some insights?

**Comment by moononournation** [comment, marked as answer]
try add 2.2k resistor in the middle of signal pins like this:
https://youtu.be/-xrbyeakgBQ

「Michael Miller <notifications@github.com>」在 2021年2月24日週三，07:46 寫道：

**Reply by Makuna** [reply]
That fixed it!  
So the pins are not 5v tolerant and cause this strange side effect.

**Comment by moononournation** [comment]
yes, same as ILI9341

「Michael Miller <notifications@github.com>」在 2021年2月24日週三，08:04 寫道：

**Reply by tcerdaj** [reply]
My greatest respect to you and thanks for sharing such valuable information in this forum. I am currently trying to configure the wiring for Arduino Mega using the GC9A01 module, if possible could you share the diagram, please? Thank you.

I am trying https://www.waveshare.com/wiki/1.28inch_LCD_Module documentation, but they only have it for Arduino uno.

**Reply by moononournation** [reply]
As mentioned in instructables, ILI9341 and GC9A01 require extra effort for 5V MCU. It is recommended using 3.3V MCU for this display.
